spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -2690,7 +2686,7 @@ def _extract_iobufs(self):
         extractor = ChipIOBufExtractor()
         extractor(
             transceiver=self._txrx,
-            core_subsets=self._last_run_outputs["CoresToExtractIOBufFrom"],
+            executable_targets=self._last_run_outputs["ExecutableTargets"],
```

Comment:
this has resulted in the loss of the iobuf control flags in cfg. will need to be reengineered

Reply:
fixed!